src/core/state/hiddenAssets/hiddenAssets.ts

```diff
@@ -26,15 +27,39 @@ export const hiddenAssetsStore = createStore<HiddenAssetState>(
     hidden: {},
     toggleHideAsset: (address: Address, uniqueId: string) => {
       const { hidden } = get();
+      const wasHidden = hidden[address]?.[uniqueId] || false;
       set({
         hidden: {
           ...hidden,
           [address]: {
             ...(hidden[address] ?? {}),
-            [uniqueId]: !hidden[address]?.[uniqueId],
+            [uniqueId]: !wasHidden,
           },
         },
       });
+      const updatedState = get();
+      const isNowHidden = updatedState.hidden[address]?.[uniqueId] || false;
+      const totalHiddenForAddress = Object.values(
+        updatedState.hidden[address] || {},
+      ).filter((isHidden) => isHidden).length;
+
+      const [assetAddress, chainIdStr] = uniqueId.split('-');
+
+      const assetHiddenAnalytics = {
+        token: {
+          address: assetAddress,
+          chainId: parseInt(chainIdStr),
+          walletAddress: address,
```

Comment:
idk if this is still the case, but are we sure we want to pass this data across for analytics? IIRC wallet addresses were treated as sensitive data, and this would expose a lot of tracking that could be deemed as not so savory.

Reply:
yeah probably unnecessary tbh. i can remove.